docker/localnode/config/app.toml

```diff
@@ -0,0 +1,203 @@
+# This is a TOML config file.
```

Comment:
i dont think we need this file - app.toml is automatically generated by `seid init`

Reply:
on 2nd thought, might be nice to keep it so we can make changes locally and run the script rather than ssh'ing into each container and editing the file